packages/design-system-web/src/components/geov-yasgui-data-validation/geov-yasgui-data-validation.tsx

```diff
@@ -10,6 +10,12 @@ interface ExpectedKey {
 /**
  * The component has to validate input data and emit the validation result (valid/invalid).
  *
+ * It requires two inputs: 'data' (the input data to be validated) and 'expectedKeys' (an array of expected keys).
+ *
+ * The component emits a custom event named "validationCompleted" with a boolean value:
```

Comment:
almost, sorry, I know this is pedantic. But it's helpful that they have to check the _isValid_ property.
```suggestion
 * The component emits a custom event named "validationCompleted" with a custom boolean event property "isValid":
```

Reply:
No problem, thank you !